pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -566,6 +566,10 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 			glog.V(4).Infof("Stopping PodSandbox for %q, will start new one", format.Pod(pod))
 		}
 
+		if podStatus == nil {
```

Comment:
Won't the first line (`m.computePodContainerChanges(pod, podStatus)`) fail if podStatus is `nil`?